sources/packages/web/src/views/institution/WithdrawalUpload.vue

```diff
@@ -0,0 +1,316 @@
+        <li>
+          If your file has errors, please fix it in the excel file first and
+          re-upload
+        </li>
+        <li>Once there are no errors, click the "Create now" button</li>
+      </ul>
+      <p class="category-header-medium primary-color">Additional notes</p>
+      <ul class="m-4">
+        <li>
+          This will process each record as if it were a manually entered
+          withdrawal
+        </li>
+        <li>
+          Any errors will produce a list below. These errors must be fixed in
+          order to bulk upload your withdrawal file
+        </li>
+        <li>
+          When clicking "Validate" any warnings will produce a list below.
+        </li>
+      </ul>
+      <v-divider />
+      <v-form ref="uploadForm">
+        <v-row class="m-0 p-0">
+          <v-file-input
+            :loading="loading"
+            :clearable="true"
+            :accept="ACCEPTED_FILE_TYPE"
+            density="compact"
+            v-model="withdrawalFiles"
+            label="Withdrawal text file"
+            variant="outlined"
+            prepend-icon="fa:fa-solid fa-file-text"
+            :rules="[fileValidationRules]"
+            :key="textFileUploadKey"
+          />
+
+          <v-btn
+            class="ml-2"
+            color="primary"
+            prepend-icon="fa:fa-solid fa-file-circle-question"
+            @click="uploadFile(true)"
+            :loading="validationProcessing"
+            :disabled="loading"
+          >
+            Validate
+          </v-btn>
+          <v-btn
+            class="ml-2"
+            color="primary"
+            prepend-icon="fa:fa-solid fa-upload"
+            :loading="creationProcessing"
+            :disabled="loading"
+            @click="uploadFile(false)"
+          >
+            Create and upload bulk withdrawal file
+          </v-btn>
+        </v-row>
+        <v-progress-linear
+          v-if="loading"
+          class="mb-4"
+          rounded
+          :max="uploadProgress.total"
+          :model-value="uploadProgress.loaded"
+          color="primary"
+        />
+        <banner
+          v-if="showPossibleFileChangeError"
+          class="mb-2"
+          :type="BannerTypes.Error"
+          summary="An error occurred during the file upload. Reasons can include a file modification after the file was already selected or an unexpected network failure. Please select the file and try again."
+        />
+      </v-form>
+      <content-group v-if="validationResults.length">
+        <body-header
+          title="Validation summary"
+          :recordsCount="validationResults.length"
+        ></body-header>
+        <banner
+          class="mb-2"
+          v-if="hasCriticalErrorsRecords"
+          :type="BannerTypes.Error"
+          summary="Error! We found problems that will prevent the bulk withdrawal from being created and uploaded. Please review the errors below."
+        />
+        <content-group>
+          <DataTable
```

Comment:
just a question to devs, since its a new file, do we need to use, vuetify data table?

Reply:
Yes as per our agreement.